src/physics/em/BetheHeitlerModel.cc

```diff
@@ -57,12 +58,18 @@ void BetheHeitlerModel::interact(
     CELER_MAYBE_UNUSED const ModelInteractRefs<MemSpace::device>& pointers) const
 {
 #if CELERITAS_USE_CUDA
-    detail::bethe_heitler_interact(interface_, pointers);
+    generated::bethe_heitler_interact(interface_, pointers);
 #else
     CELER_ASSERT_UNREACHABLE();
 #endif
```

Comment:
Since we define the `inline generated::bethe_heitler_interact` inside an `#if !CELERITAS_USE_CUDA`, we can simplify all these model calls by simply writing
```c++
    generated::bethe_heitler_interact(interface_, pointers);
```

We *should* add `CELER_ASSERT_UNREACHABLE()` to the generator script inside that inline function, though (currently it is not there).